events.go

```diff
@@ -1,7 +1,7 @@
 package csminify
 
 import (
-	rep "github.com/markus-wa/cs-demo-minifier/replay"
+	rep "github.com/markus-wa/cs-demo-minifier/replay" 
```

Comment:
![image](https://user-images.githubusercontent.com/5138316/83359301-54596c80-a371-11ea-9e36-96f64721b176.png)

Reply:
:D

Reply:
it can! :slightly_smiling_face: with the help of the 'Save Actions' plugin:

![image](https://user-images.githubusercontent.com/5138316/83360323-4529ed00-a378-11ea-8357-0badada79dd4.png)

Reply:
thanks :)